### ISSUE TITLE
Unable to get operator's information from traced graph node

### ISSUE THREAD

**Issue by xiayouran** [pytorch/pytorch#81099]
### 🐛 Describe the bug

**Unable to get operator's information from traced graph node.**

In version 1.3.1, I can see the details of nodes, such as convolution. I can know the information of convolution: **kernel size, padding, stride**, but in version 1.9.1, I can't. How can I get detailed operator information in version 1.9.1

```python
import torch
from torchvision import models

model = models.vgg16()
dummy_input = torch.rand(1, 3, 224, 224)

# in 1.3.1
with torch.onnx.set_training(model, False):
    trace, _ = torch.jit.get_trace_graph(model, dummy_input, _force_outplace=True)
    torch.onnx._optimize_trace(trace, operator_export_type=torch.onnx.OperatorExportTypes.ONNX)
    trace_inlined_graph = trace.graph()
    for node in trace_inlined_graph.nodes():
        print(node)

# in 1.9.1
with torch.onnx.select_model_mode_for_export(model, torch.onnx.TrainingMode.EVAL):
    trace = torch.jit.trace(func=model, example_inputs=dummy_input, _force_outplace=True)
    torch.onnx._optimize_trace(trace, operator_export_type=torch.onnx.OperatorExportTypes.ONNX)
    # trace_graph = trace.graph
    trace_inlined_graph = trace.inlined_graph
    for node in trace_inlined_graph.nodes():
        print(node)
```

### Versions

**in 1.3.1**
![image](https://user-images.githubusercontent.com/45480822/177960461-98d3c83a-cd52-440e-b627-9a5a747b75a9.png)

**in 1.9.1**
![image](https://user-images.githubusercontent.com/45480822/177959553-f9839943-b27d-46f8-8b10-26a4f5edd219.png)
